Core/src/main/java/com/plotsquared/core/util/RegionManager.java

```diff
@@ -208,9 +245,26 @@ public void swap(Location pos1, Location pos2, Location swapPos, final Runnable
         copyFromTo(pos1, pos2, relX, relZ, world1, fromQueue1, toQueue2, true);
         copyFromTo(pos1, pos2, relX, relZ, world1, fromQueue2, toQueue1, true);
         fromQueue1.setCompleteTask(fromQueue2::enqueue);
+        if (actor != null && Settings.QUEUE.NOTIFY_PROGRESS) {
+            fromQueue1.addProgressSubscriber(subscriberFactory.create(actor, taskManager, Settings.QUEUE.NOTIFY_INTERVAL, Settings.QUEUE.NOTIFY_WAIT,
+                StaticCaption.of("<prefix><gray>Current region 1 copy progress: </gray><gold><progress></gold><gray>%</gray>")));
+        }
```

Comment:
Why is this a static caption? This could be moved into a TranslatableCaption and append `<progress>%` as template.
(Same applies to all in this file).